node-express-react/node-express/.editorconfig

```diff
@@ -0,0 +1,40 @@
+root = true
```

Comment:
Whoa, I'm surprised I haven't seen this config before. Should we put this in root of the whole project so it's shared between all the examples?

Reply:
mm, good questio. I would say no, it no longer makes sense to do this.